atm.js

```diff
@@ -1,21 +1,87 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+// As a user, I want to deposit money into one of the bank accounts
+var balanceDisplay = $(".account").children().eq(1); // Selects Checking Balance HTML Area
+var userInputChecking = $(".account").children().eq(2); // Selects Checking User Input
+var checkDepositButton = $(".account").children().eq(3); // Checking Deposit Button
+var checkWithdrawlButton = $(".account").children().eq(4); // Checking Withdrawl Button
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var balanceDisplaySavings = $(".account").children().eq(6); // Selects Savings Balance HTML Area
+var userInputSavings = $(".account").children().eq(7); // Selects Savings User Input
+var savingsDepositButton = $(".account").children().eq(8); // Savings Deposit Button
+var savingsWithdrawlButton = $(".account").children().eq(9); // Savings Withdrawl Button
```

Comment:
I like how you defined all of your UI variables up at the top, and in a neat and organized way! One small suggestion... since you can edit this HTML, why not add ids or classes so that you can use those in your selectors instead of `.children().eq(x)`; this would make your code more readable, and less likely to break if the HTML is changed.

e.g. `$(".account button#deposit-btn")`